website/content/docs/builders/index.mdx

```diff
@@ -7,12 +7,16 @@ page_title: Builders
 
 # Builders
 
-Builders are responsible for creating machines and generating images from them
-for various platforms. For example, there are separate builders for EC2,
-VMware, VirtualBox, etc. Packer comes with many builders by default, and can
-also be extended to add new builders. See [`plugins`](/plugins) for a list of plugins and their builders.
+Builders create machines and generate images from those machines for various platforms. Packer also has some builders that perform helper tasks, like running provisioners.
+
+Packer has the following types of builders:
+
+- [File](/docs/builders/file): The `file` builder creates an artifact from a file.
+- [Null](/docs/builders/null): The `null` builder sets up an SSH connection and runs the provisioners.
+- [Plugin](/plugin): Each plugin has its own associated set of builders. For example, there are separate builders for EC2, VMware, VirtualBox, etc.
```

Comment:
I would move this first since it's the most common/important one

Reply:
done - ty!